api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSourceProvider.java

```diff
@@ -44,8 +44,11 @@
  */
 public interface ConfigSourceProvider {
     /**
-     * Return the {@link ConfigSource} instances that are provided by this provider. An empty {@code Iterable} may
-     * be returned if no sources are to be provided.
+     * Return the {@link ConfigSource} instances that are provided by this provider.  
+     * The {@link java.util.Iterator Iterable} contains a fixed number of {@linkplain ConfigSource configuration sources}, 
```

Comment:
Did you mean to link to `Iterable`?  This links to `Iterator`.

Reply:
well spoted. fixed.